packages/kit/types/index.d.ts

```diff
@@ -323,6 +324,7 @@ export interface ServerInitOptions {
 
 export interface SSRManifest {
 	appDir: string;
+	prefix: string;
```

Comment:
`prefix` is pretty vague/generic. How about `app_path` to match the suggestion above? https://github.com/sveltejs/kit/pull/4448/files#r984006863

Reply:
Changed to `appPath` and changed builder.prefix to `builder.app_path` to match